schemas/com.vimeo/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,90 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo video.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "video",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "videoId": {
+      "type": "number",
+      "description": "The Vimeo ID of the video.",
+      "minimum": 0,
+      "maximum": 9223372036854776000
+    },
+    "videoTitle": {
+      "type": "string",
+      "description": "The title of the video.",
+      "maxLength": 4096
+    },
+    "videoUrl": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "format": "uri",
+      "description": "The URL of the video on vimeo.com.",
+      "maxLength": 4096
+    },
+    "videoWidth": {
+      "type": "number",
+      "description": "The native width of the video as the width of the video's highest available resolution.",
+      "minimum": 0,
+      "maximum": 9223372036854776000
+    },
+    "videoHeight": {
+      "type": "number",
+      "description": "The native height of the video as the height of the video's highest available resolution.",
+      "minimum": 0,
+      "maximum": 9223372036854776000
+    },
+    "error": {
```

Comment:
There is a new [error event](https://github.com/snowplow/iglu-central/pull/1283/commits/24d241c5de52bae81c048e596117f8daf6d9b0a0) which contains the error code and description properties. Not sure if we can use that for the info here, just mentioning for you to consider.

Reply:
Removed in favour of the new error event.